src/appleseed/renderer/kernel/lighting/sppm/sppmparameters.cpp

```diff
@@ -96,7 +96,8 @@ namespace
 }
 
 SPPMParameters::SPPMParameters(const ParamArray& params)
-  : m_sampling_mode(get_sampling_context_mode(params))
+  : m_spectrum_mode(get_spectrum_mode(params))
```

Comment:
Right now, we are collecting all color related parameters in the Frame entity.
Would it make sense to move spectrum_mode there too?